component/common/pom.xml

```diff
@@ -93,6 +93,12 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>@{argLine} ${env.MAVEN_OPTS} --add-opens=java.base/jdk.internal.vm=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED --add-opens=java.base/java.security=ALL-UNNAMED</argLine>
```

Comment:
May be to add here https://github.com/Meeds-io/maven-parent-pom/blob/38f428610ced8e1824eb72d9dd35c5635b771fc8/pom.xml#L768 WDYT ?

Reply:
We may need to add those options here as well (for runtime):
https://github.com/Meeds-io/meeds/blob/88622dd72c45a772a792311350c15c606cb0c09d/packaging/plf-tomcat-resources/src/main/resources/bin/setenv.sh#L293